pyactiveresource/activeresource.py

```diff
@@ -537,19 +538,19 @@ def _build_object(cls, attributes, prefix_options=None):
         return cls(attributes, prefix_options)
 
     @classmethod
-    def _build_list(cls, elements, prefix_options=None):
-        """Create a list of objects for the given xml string.
+    def _build_collection(cls, elements, prefix_options=None, query_options={}):
+        """Create a collection of objects for the given xml string.
 
         Args:
             elements: A list of dictionaries representing resources.
             prefix_options: A dict of prefixes to add to the request for
                             nested URLs.
         Returns:
-            A list of ActiveResource objects.
+            A collection of ActiveResource objects.
         """
-        resources = []
+        resources = Collection(elements)
         if isinstance(elements, dict):
-          elements = [elements]
+          elements = Collection()[elements]
```

Comment:
This seems wrong, this will raise an exception if the response is a dictionary. Should be `Collection([elements])`

Reply:
Hmm, that would be wrong, since it's not used as a collection. Just `[elements]` like it originally was should suffice.